### PR TITLE
Add delete member feature

### PR DESCRIPTION
**Pull request by Kajol-Kumari**
#2648 

Participant can now remove it's member whenever needed instead of creating a new participant team from scratch.

### CODE REVIEW

**Comment by RishabhJain2018** [comment]
@Kajol-Kumari Can you please fix the Travis build?

**Comment by Kajol-Kumari** [comment]
> @Kajol-Kumari Can you please fix the Travis build?

Just working on it @RishabhJain2018

File: apps/participants/views.py [hunk 1]
```diff
@@ -278,9 +278,11 @@ def delete_participant_from_team(request, participant_team_pk, participant_pk):
     except ParticipantTeam.DoesNotExist:
         response_data = {"error": "ParticipantTeam does not exist"}
         return Response(response_data, status=status.HTTP_406_NOT_ACCEPTABLE)
+    user = User.objects.get(id=participant_pk)
 
     try:
-        participant = Participant.objects.get(pk=participant_pk)
+        participant = Participant.objects.get(user=user,team=participant_team)
+        print(participant)
```
Comment (vkartik97) [inline apps/participants/views.py]: Can you please remove this line?

**Comment by Kajol-Kumari** [comment]
@RishabhJain2018 @vkartik97 can you please help me to figure out why am i unable to get a particular paticipant by using participant_pk as written the previously written code?

**Comment by Ram81** [comment]
@Kajol-Kumari issue is with flake8. You can see in the travis CI logs the issue is this
```
./apps/participants/views.py:284:56: E231 missing whitespace after ','
The command "flake8 ./" exited with 1.
```

**Comment by Sanji515** [comment]
@Kajol-Kumari I think you also need to update the tests [here](https://github.com/Cloud-CV/EvalAI/blob/master/tests/unit/participants/test_views.py#L576) as you have modified the API `delete_participant_from_team`.

**Comment by Kajol-Kumari** [comment]
> @Kajol-Kumari issue is with flake8. You can see in the travis CI logs the issue is this
> 
> ```
> ./apps/participants/views.py:284:56: E231 missing whitespace after ','
> The command "flake8 ./" exited with 1.
> ```

corrected it, Thanks!

**Pull request closed without merging**